blocks/library/code/index.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * Internal dependencies
+ */
+import './style.scss';
+import { registerBlock, query } from '../../api';
+import Editable from '../../editable';
+
+const { children } = query;
+
+registerBlock( 'core/code', {
+	title: wp.i18n.__( 'Code' ),
+
+	icon: 'editor-code',
+
+	category: 'common',
+
+	attributes: {
+		content: children( 'code' ),
+	},
+
+	edit( { attributes, setAttributes, focus, setFocus } ) {
+		return (
+			<Editable
```

Comment:
I guess we don't have to pass a focus object. Just noticing that the texture is not focussing immediately on click.